arxiv/avoiding_catastrophe/active_dendrites_enable_multitask_learning/notebooks/analyze_network_vlatest.ipynb

```diff
@@ -0,0 +1,712 @@
+     },
+     "metadata": {
+      "needs_background": "light"
+     }
+    }
+   ],
+   "source": [
+    "state_path = \"/Users/lsouza/nta/results/no_preprocess3-wouv839/experiment_state.p\"\n",
```

Comment:
You may want to replace this path with something relative to this repo

Reply:
For safety? The notebooks are there not to be runned, just for visualization. I added a readme under the notebook folders that says that. The code in all notebooks is not gonna work under the current version.